pkg/build/pipelines/ecosystems/maven.yaml

```diff
@@ -0,0 +1,69 @@
+name: Configure maven to fetch Chainguard Libraries dependencies.
+
+needs:
+  packages:
+    - busybox
+
+inputs:
+  repository:
+    description: Maven repository to use
+    default: "https://libraries.cgr.dev/java-all/"
+
+pipeline:
+  - name: Setup Maven credentials
+    runs: |
+      set -euo pipefail
+
+      # If libraries token exists in melange's cache, use that.
+      # Fetch the token with: make lib-token
+      if [ ! -f /var/cache/melange/.libraries_token.txt ]; then
+        echo "Failed to find libraries token. Please generate with: make lib-token"
+        exit 1
+      fi
+
+      echo "Removing local maven repository, if exists..."
+      rm -rf /root/.m2 && mkdir -p /root/.m2
+
+      echo "Creating maven settings manifest..."
+      cat > /root/.m2/settings.xml <<EOF
```

Comment:
Is /root/ a safe assumption? I'm not up to speed on which melange runner(s) hardcode which users and paths and stuff. Don't change the PR based on this comment, just asking in general.